packages/models-library/src/models_library/projects.py

```diff
@@ -17,13 +18,24 @@
 from .projects_state import ProjectState
 from .projects_ui import StudyUI
 
-ProjectID = UUID
-ProjectIDStr = constr(regex=UUID_RE_BASE)
+ProjectID: TypeAlias = UUID
+ClassifierID: TypeAlias = str
 
-ClassifierID = str
+NodesDict: TypeAlias = dict[NodeIDStr, Node]
```

Comment:
Why this name change? this is still the Workbench right? can we keep it so? or is there a valid reason for changing the name?

Reply:
Yes, it is still the workbench. Before the entry was
```python

class Project(BaseModel):
    # ...
    workbench: Workbench
    # ...
```

and now is 
```python

class Project(BaseModel):
    # ...
    workbench: NodesDict
    # ...
```

which IMO is much more descriptive